calendar-bundle/contao/modules/ModuleEventReader.php

```diff
@@ -158,6 +161,28 @@ protected function compile()
 			{
 				$htmlHeadBag->setMetaRobots($objEvent->robots);
 			}
+
+			if ($objEvent->canonicalLink)
```

Comment:
```suggestion
			if ($objCalendar->enableCanonical && $objEvent->canonicalLink)
```

The link should only be added if `$objCalendar->enableCanonical` is true, shouldn‘t it? At least that is how we do it for pages.